examples/with-firebase-authentication/utils/middleware/cookieSession.js

```diff
@@ -3,19 +3,22 @@ import cookieSession from 'cookie-session'
 export const addSession = (req, res) => {
   // Ensure that session secrets are set.
   if (
-    !(process.env.SESSION_SECRET_CURRENT && process.env.SESSION_SECRET_PREVIOUS)
+    !(
+      process.env.NEXT_PUBLIC_SESSION_SECRET_CURRENT &&
+      process.env.NEXT_PUBLIC_SESSION_SECRET_PREVIOUS
```

Comment:
I believe the keys is now called as before

Reply:
Sorry, i missed them!